vulns/CVE-2024-57841.yml

```diff
@@ -0,0 +1,7 @@
+reachability: Local
```

Comment:
We had evaluated this as remotely-reachable, because the affected function processes incoming TCP packets. Did you have reasons to believe this isn't the case? I will actually push our assessment of this vuln soon, happy to rework it if it's inaccurate.

Reply:
My bad, my assumption was flawed in the sense that it is the user who starts off the process.
I agree, to the explanation and will push out a change to fix it.